NuKeeper.GitHub/OctokitClient.cs

```diff
@@ -59,7 +59,12 @@ public async Task<User> GetCurrentUser()
             var user = await _client.User.Current();
             var userLogin = user?.Login;
             _logger.Detailed($"Read github user '{userLogin}'");
-            return new User(user.Login, user.Name, user.Email);
+            if (user != null)
+            {
+                return new User(user.Login, user.Name, user.Email);
```

Comment:
can't you just do `return new User(user?.Login, user?.name, user?.Email)` and delete the if?

Reply:
That would return a non-null user in every case